### PR TITLE
Query Multiple Tables

### PR DESCRIPTION
**Pull request by aamir-asaram**
**In this PR, I've implemented the following:**

Added to `schema.sql`:
- Created `owners` and `species` tables
- Removed the `species`  column from the animals table and replaced it with `owner_id` and `species_id`referencing the `owner` and `species` tables respectively

Added to `data.sql`:
- Added sample data to `owners` and `species` tables
- Modified the `animals` table to include the `species_id` from the `species` table as well as the `owner_id` from the `owners` table

Added to `queries.sql`: 
- Various multi-table queries using different `join` functions as shown below:

query multiple tables

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,36 +6,26 @@ After you're finished please remove all the comments and instructions!
...
-
-> Describe your project in 1 or 2 sentences.
+# 📖 Vet Clinic <a name="about-project"></a>
 
-**[your_project__name]** is a...
+Vet Clinic is a database application used to keep track of the various animals visiting a vet.
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
```
Comment (Lucas-Masaba) [inline README.md:27]: - [x] You added your client as React but you are not using React in this project. Please remove this.
- [x] You added your server as Express but it is not being used in this project. Please remove it.

Please ensure to have good and clear documentation so as not to confuse your users.

File: README.md [hunk 2]
```diff
@@ -260,23 +126,7 @@ If you like this project...
 
 > Give credit to everyone who inspired your codebase.
```
Comment (Lucas-Masaba) [inline README.md]: - [x] Please remove this comment. It is from the default README and it is just provided information on what a student should write under this header. Please do the same for the "Show your support" header.

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @aamir-asaram  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Maged to query multiple tables :+1: 
- SQL entered in there respective files :file_cabinet: 

## Required Changes ♻️
_Check the comments under the review._

- [ ] Please add screenshots showing all your query results. The image you shared cuts out some results :ok_hand: 

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by aamir-asaram** [comment]
Thank you for the helpful suggestions! @Lucas-Masaba

**Review by divinecharlotte — APPROVED** [review]
# Hello @aamir-asaram   😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say  👍🏻 ✅ 
### To Highlight 💯


- Good implementation of the required changes 👍 
- clean code ✔ 👍
 




## Congratulations! 🎉 



Cheers, and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**